README.md

```diff
@@ -10,6 +10,40 @@ Description
 
 Feature Highlights
 
+* **Create articles dynamically with paragraphs**: Using paragraphs, you can add text, pictures, videos, Instagram or Twitter cards to your article – and change the order by dragging and dropping the content wherever you like it. Thanks to a preview mode, you get a good idea of how your article will look like in the end.
+* **Working in distributed teams**: Thunder is a web-based system, so you can add and edit articles from everywhere. All you need is a computer and access to the Internet. Every modification to an article is saved in revisions. So, you can easily understand who did changes to your article. Plus, you can share a link to an unpublished article with external co-workers and give them access for a limited time. 
+* **User management**: In Thunder, you can define lots of different roles for every need—and allocate various rights.
+* **Liveblog**: This module helps to cover events as they happen, like sport events or elections.
```

Comment:
Its not yet integrated, and will not be released before drupal 8.3 is released in April. So maybe we should not have it in the featurelist as of now.